lib/engine-addon.js

```diff
@@ -387,10 +391,76 @@ module.exports = {
         return mergeTrees([
           externalTree,
           engineJSTree,
-          engineCSSTree
+          compiledEngineCSSTree
         ].filter(Boolean), { overwrite: true });
       };
 
+      this.compileLazyEngineStyles = function compileLazyEngineStyles(vendorTree, externalTree) {
+        var vendorCSSTree = buildVendorCSSTree.call(this, vendorTree);
+        var engineStylesOutputDir = 'engines-dist/' + this.name + '/assets/';
```

Comment:
I don't really think this changes much, I'd prefer to duplicate than pull in a dep and join.